sync_test.go

```diff
@@ -379,21 +372,9 @@ func testRescan(harness *neutrinoHarness, t *testing.T) {
 		t.Fatalf("Couldn't sync ChainService: %s", err)
 	}
 
-	// Call GetUtxo for our output in tx1 to see if it's spent.
-	ourIndex := 1 << 30 // Should work on 32-bit systems
-	for i, txo := range tx1.TxOut {
-		if bytes.Equal(txo.PkScript, script1) {
-			ourIndex = i
-		}
-	}
-	if ourIndex != 1<<30 {
-		ourOutPoint = wire.OutPoint{
-			Hash:  tx1.TxHash(),
-			Index: uint32(ourIndex),
-		}
-	} else {
-		t.Fatalf("Couldn't find the index of our output in transaction"+
```

Comment:
From what I could tell, there was no index search required, since we are constructing the transaction ourselves just a few lines up, and there is not output shuffling in the `CreateTransaction` path. Which is all for the better, because I wanted an output at a non-zero index to check that the field was being populated correctly.